docs/source/examples/workspace/PinFile.complex-new.yml

```diff
@@ -0,0 +1,90 @@
+---
```

Comment:
What is the purpose of creating a new pinfile? Can you put these into the existing ones? PinFiles here are generally for the purpose of testing and documentation. Adding/changing them should be to make more tests. Do we need a new pinfile for this functionality? Or can you use an existing one without modification?

Reply:
I initially needed to be able to demonstrate validation on a PinFile with only new topologies, but I don't think that's necessary anymore